src/xcode/ENA/ENA.xcodeproj/project.pbxproj

```diff
@@ -424,6 +424,7 @@
 		130CB19B246D92F800ADE602 /* ENAUITestsOnboarding.swift */ = {isa = PBXFileReference; fileEncoding = 4; lastKnownFileType = sourcecode.swift; path = ENAUITestsOnboarding.swift; sourceTree = "<group>"; };
 		13156CFE248C19D000AFC472 /* de */ = {isa = PBXFileReference; lastKnownFileType = text.html; name = de; path = de.lproj/usage.html; sourceTree = "<group>"; };
 		13156D00248CDECC00AFC472 /* en */ = {isa = PBXFileReference; lastKnownFileType = text.html; name = en; path = en.lproj/usage.html; sourceTree = "<group>"; };
+		1347D58924B7753C0028E102 /* ar */ = {isa = PBXFileReference; lastKnownFileType = text.html; name = ar; path = "ar.lproj/privacy-policy.html"; sourceTree = "<group>"; };
```

Comment:
This looks odd.

There is a line added for `ar.lproj/privacy-policy.html`, but there isn't a line added for
```
ar.lproj/usage.html
ru.lproj/privacy-policy.html 
ru.lproj/usage.html 
```

Reply:
@i336616 can you check whether some reference is missing? :)

Reply:
Re-added TR (turkish) localization references back into project.